examples/cpp/src/cagra_c_example.cu

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2022-2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cstdint>
```

Comment:
Please put this in `examples/c` so that it's more straightforward to users. Also, please make it self-contained so that C users don't need to try and pull it apart from the C examples.

Reply:
What kind of dataset can I use for a self-contained example? Is a small, hardcoded dataset okay?